testing/examples/author_tests.rs

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Are these now tests or examples :D? I suggest that we stay consistent with the naming. We can go either for tests or examples, but I am against this schizophrenic scheme.

Reply:
Actually i think we should either make it bins, then we don't need to add the `--example` flag all the time, what do you think?

Reply:
The good thing about examples is that they are built automatically with `cargo test` (but not run). 

Binaries are also possible. Implication: Binaries are built with normal "cargo build --release". Not sure if that's a drawback or not.
What's certainly a drawback: CI needs to be adapted quite a bit, because the binaries are stored in the `release` folder. Or is there a way to define an output folder?

Reply:
Nope, the change would imply that we need to copy it from another folder. Hmm, it is hard to say.

Ok, I will back off. If you think this is the suitable solution, I happily go with it.

Reply:
Suitable... not sure. But the simplest one at leat 😄